benchmarks/starlette/run.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+exec gunicorn app:app --workers $WORKERS --threads $WORKERS --config gunicorn_conf.py
```

Comment:
If worker is uvicorn, why don't we simply use the uvicorn executable ?

Reply:
I followed the documentation: https://www.starlette.io/#performance maybe there is a better/simpler way?

Reply:
OK, I'd say let's follow the doc :)